OrbitBase/CMakeLists.txt

```diff
@@ -31,24 +32,35 @@ target_sources(OrbitBase PRIVATE
 
 target_sources(OrbitBase PRIVATE
         Logging.cpp
+        Path.cpp
+        SafeStrerror.cpp
         ThreadPool.cpp
-        Tracing.cpp
-        SafeStrerror.cpp)
+        Tracing.cpp)
+
+if (WIN32)
+target_sources(OrbitBase PRIVATE
+        PathWindows.cpp)
+else()
+target_sources(OrbitBase PRIVATE
+        PathLinux.cpp)
+endif()
 
 target_link_libraries(OrbitBase PUBLIC
         CONAN_PKG::Outcome
         CONAN_PKG::abseil
         std::filesystem)
 
+
 add_executable(OrbitBaseTests)
 
 target_compile_options(OrbitBaseTests PRIVATE ${STRICT_COMPILE_FLAGS})
 
 target_sources(OrbitBaseTests PRIVATE
-    UniqueResourceTest.cpp
+    PathTest.cpp
```

Comment:
Move down to keep the alphabetical order.

Reply:
done